cspell.yml

```
@@ -0,0 +1 @@
+words: [commitlint, schemars, commitlintrc, Keke, insta, binstall, serde, Keisuke, astrojs, splitn, insta, schemars]
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)